test/unit/dependency_resolver_test.rb

```diff
@@ -9,15 +9,24 @@
   describe "for resource" do
 
     it "returns empty array for an independent resource" do
-      resource = HammerCLIForeman.foreman_resource!(:architectures)
+      resource = HammerCLIForeman.foreman_resource!(:bookmarks)
       resolver.resource_dependencies(resource).must_equal []
     end
 
     it "returns list of dependent resources" do
       resource = HammerCLIForeman.foreman_resource!(:images)
-      resolver.resource_dependencies(resource).map(&:name).sort_by{|sym| sym.to_s}.must_equal [
-        :compute_resources, :organizations, :locations
-      ].sort_by{|sym| sym.to_s}
+      resources = resolver.resource_dependencies(resource).map(&:name).sort_by{ |sym| sym.to_s }
+      if FOREMAN_VERSION < Gem::Version.new('1.10')
```

Comment:
Since you added `test/unit/data/test_api.json` I propose to switch this test to use the fake data too. What do you think?